zombienet_tests/functional/0001-parachains-pvf.toml

```diff
@@ -117,7 +117,7 @@ genesis_state_generator = "undying-collator export-genesis-state --pov-size=1000
   name = "collator07"
   image = "{{COL_IMAGE}}"
   command = "undying-collator"
-  args = ["-lparachain=debug", "--pov-size=100000",  "--pvf-complexity=300", "--parachain-id=2006"]
+  args = ["-lparachain=debug", "--pov-size=100000",  "--pvf-complexity=300", "--parachain-id=2006", "--hrmp-params=2007:4096", "-lruntime::undying=debug"]
```

Comment:
I would create a separate test for this where we match on some channel utilization metrics to be sure everything works as expected.

To make this truly deterministic we would need to be able to configure the test parachain to send a predefined amount of messages then stop. We can then expect this to happen in a number of relay chain blocks - a minimum tput.